pkgs/applications/radio/flamp/default.nix

```
@@ -0,0 +1,67 @@
+{ lib
+, stdenv
+, fetchgit
+, autoreconfHook
+, pkg-config
+, hamlib
+, fltk13
+, libjpeg
+, libpng
+, portaudio
+, libsndfile
+, libsamplerate
+, libpulseaudio
+, libXinerama
+, gettext
+, alsa-lib
+, udev
+}:
+
+stdenv.mkDerivation rec {
+  pname = "flamp";
+  version = "2.2.07";
+
+  src = fetchgit {
+    url = "https://git.code.sf.net/p/fldigi/flamp";
+    rev = "v2.2.07";
+    sha256 = "sha256-BnKL1iwaw1iJC5qcGCvdMr461n0BSYxk61dbAnhpz2c=";
+  };
+
+  nativeBuildInputs = [ autoreconfHook pkg-config ];
```

```suggestion
  nativeBuildInputs = [
    autoreconfHook
    pkg-config
  ];
```
One item per line is preferred.